### ISSUE TITLE
Lead is painfully scarce

### ISSUE THREAD

**Issue by ubertpendragon** [Regalis11/Barotrauma#10613]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Almost every single ammunition recipe competes Lead. 

Coilguns, Railguns, Chainguns... but also HMG, Assault Rifle, Rifle, SMG, Shotgun, Handcanon, Revolver...
It's actually easier to make piercing and depleted coilgun ammunition because it doesn't use Lead

In the Factions session we had to switch a double coilgun for a flak only because flak ammunition doesn't use lead (it's super cheap too)

Some ammunition types need to be rebalanced to use not use Lead, and use more cheaper alternatives instead.
More specifically:

-Revolver ammo costs more lead (and magnesium) than a single HMG magazine. Use zinc instead of Lead
-SMG mag costs the same amount of Lead as AR mag, but is weaker. Replace 1x Lead with 2x Copper
-Handcannon rounds use 2x Lead for 6 bullets. If Revolver ammo is cheaper, reduce cost to 1x Lead
-Shotgun shells use 2x Lead, reduce to 1x

### Bug prevalence

Happens every time I play

### Version

0.20.8.0 (Unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by aeondez** [comment]
All the ammo recipes using lead makes sense.

But if every kind of ammo needs lead, the availability of lead should be like fuel rods, welding fuel, screwdrivers, and wrenches. Available everywhere, and relatively inexpensive.

**Comment by BiffW** [comment]
Making physicorium ammo box no longer require lead to craft could also help.

**Comment by Kaktar-1** [comment]
I've never ran out of lead before. Even on the hardest difficulty with Jovian radiation enabled, we ran out of magnesium but not lead. Using weapons such as _harpoons and melee to deal with weaker and slower enemies_ and balancing the ship's weapons to _avoid using too much of one resource for ammunition_ helps drastically. Having a good foresight into _what routes you are taking and how low long your ammunition will last_ can also keep resources at a healthy amount. This should go without saying, but it saves a lot of ammunition if you _avoid fights you don't have to engage in_ too, such as the Ruin Guardians, Moloch's you can turn the sonar off for, and encounters your ship can outpace.
Moreover: Weaker weapons might result in more injuries, but if you happen to also find yourself frequently lacking healing items then it might be time to lower the difficulty seeing that both enemy health and damage is a problem.

I'm not saying some of the recipes containing lead don't need to be reworked, but if they were, I'd expect to see the price of lead **increased** and its mineral spawns **reduced** since it's an incredibly valuable resource given its practical uses in ammunition, fuel and advanced technologies.
Making lead more abundant with no drawbacks seems unnecessary to me since it would make resource management less important if you and your crew want to flourish. In short: **_Get youself an accountant!!!_**
I ultimately don't see any issues with lead as a resource right now even though some recipes are somewhat ridiculous.


Also, try this challenge:

- Start with the Barsuk on the hardest difficulty with Jovian radiation enabled, don't buy or fabricate items that contain lead. 

- You can recycle things that contain lead such as reactor fuel since this challenge would almost be impossible otherwise, but don't fabricate anything that consumes lead.

- You can use lead items you find on the ocean floor, pirate ships and outposts but you absolutely cannot buy or fabricate anything that uses lead. 

- It might be awhile before you and your crew get the hang of it, but at the very least you will all learn to be somewhat less dependent on lead ammunition and will develop some tactics to combat enemies when that resource is scarce.

Try and get to the end of the first two biomes doing this. It's possible and not nearly as hard as one might think.


On a closing note, cave mining at least once every three to four routes is enough to get you through the entire game.
Just skirt a route's walls and you'll eventually find one or two. It also gives bored crewmates and newbies something to do.

**Comment by T-rexsuper1** [comment]
> All the ammo recipes using lead makes sense.
> 
> But if every kind of ammo needs lead, the availability of lead should be like fuel rods, welding fuel, screwdrivers, and wrenches. Available everywhere, and relatively inexpensive.

No. No it does not. And here's why. 

Steel can be used to replace lead in practically every instance of weapon projectiles. You see, steel jacket or sometimes copper jacket is almost always present. The lead part is the core of the projectile. However, modern firearms mostly do not use lead cores and instead use steel or other materials. 
Here's an example from US military. 
"For general issue, the U.S. Army adopted the M855A1 round in 2010 to replace the M855. The primary reason was pressure to use non-lead bullets. The lead slug is replaced by a copper alloy slug in a reverse-drawn jacket, with a hardened steel penetrator extending beyond the jacket" from Wikipedia, article on 5.56x45 NATO. 
Why was it done? Mostly for environment, however, tests suggest that lead is prone to deforming mid air, tumbling the projectile and destabilizing it, causing inaccuracy. So there's no real reason to use lead in projectiles and I do not understand why developers keep adding it for their recipes. Even shotgun buckshot by most part is made of steel.

**Comment by aeondez** [comment]
Fair point.

**Comment by ubertpendragon** [comment]
> Making lead more abundant with no drawbacks is largely unnecessary since it would make weapon related resource management less important to crew survival. I ultimately don't see any issues with lead as a resource right now even though some recipes are somewhat ridiculous.

Oh absolutely I don't want resources to be overabundant, scacity of resources is a good thing, it's just that many resources compete for Lead and you therefore won't use it for some of the weaker ammo/weapon types

**Comment by Ek-Videogames** [comment]
Yeah this seemed like a possible problem back when they made it so more recipes used lead ( fuel & every ammo box ), I was wondering how long it would take anybody to notice.

An aside; the in-universe logic & balance before seemed like the entire benefit of 'coilguns' as submarine armament being that they're magnetically propelled weapons and it's ammo simpler to produce because it's just metal spears not the same thing as a cartridge. But obviously it's up to the devs if they want to change around this niche of weapon roles/balancing.

**Comment by aeondez** [comment]
Balancing an in-game economy is always tricky. 

Godspeed, devs.

**Comment by ubertpendragon** [comment]
I will also add that **Military outposts sell 15 coilgun ammo boxes**, which can be deconstructed making it the most reliable way to acquire Lead. It's a bit silly
